### DISCUSSION TITLE
messages not displayed after re-install of elementary mail on eos 7 horus

### DISCUSSION THREAD

**Discussion by AddiDub** [elementary/mail#906] [Q&A]
I am a happy returning user of elementary os. But i created a problem by removing elementary mail app through apt-get and trying to re-install it. Accounts show up immediatly, but i can only see headers, messages are not displayed.

I figured out that some of the user data had been stored to /home/user/.cache , .config, .local and .var. Some of the user data seemed to be stored in folder "evolution" so i deleted them also. I tried to do a clean install of io.elementary.mail in every way i figured, but i always end up with a broken app that is not able to display the messages. 

Buttons like reply and forward are also not working. 

Is there any way to get mail to work normal again, aside from re-installing elementary os?

**Comment by leolost2605** [comment]
This might be because you have Mail installed as a Flatpak.
To check type `flatpak info io.elementary.mail`. If you don't get an error it is indeed installed as a flatpak and if it is try uninstalling it with `flatpak uninstall io.elementary.mail` and install it as a deb package with `sudo apt install io.elementary.mail`.

Hope this helps and if it doesn't please reach out again here :)

**Reply by AddiDub** [reply]
Thank you very much for your quick reply. I used apt install. I also tried to install it from elementary AppCenter but that gives the same result. As i said i could see headers but the content of the e-mails were not displayed. Searching for similar bugs there are hints that it may be related to webkit problems. As I am a noob i did not want to fiddle too much. For this time i switched to geary to be able to work. But i want to go back to elementary mail 

My main question is: where is the storage location of the config files / user data for elementary mail? I want to attempt a clean install without any old config or user data involved.

**Reply by leolost2605** [reply]
Hmm AFAIK most of the config files are stored in /home/user/.config/evolution 
Can you maybe run Mail from terminal by executing `io.elementary.mail` and post the terminal output (messages/warnings) here?
Also sorry for the late response :(

**Reply by AddiDub** [reply]
If i execute `io.elementary.mail` in the terminal i don't get any terminal output. Mail just starts up and shows the same behavior as before - not showing the messages. System is up-to-date.  Sorry for my late reply as i have been in vacation. And thanks again for trying to help me!

**Reply by NatNajera** [reply]
Hello. I have the same trouble. So i read in 2024 some body can help us?.

Ago 6 months and any can resolve .....

**Reply by jeremypw** [reply]
See bug reports #784 and #950 for more information on this. It seems to be an upstream issue so not simple to fix without a UI redesign (see PR #973 for a possible fix).  Hopefully this will resolve itself in elementary 8 when newer libraries are used.

In the meantime the only workround I have found is to start to Reply to messages that do not display properly - they should display properly in the Reply window.
